src/main/java/de/unistuttgart/overworldbackend/controller/MinigameInputController.java

```diff
@@ -0,0 +1,29 @@
+package de.unistuttgart.overworldbackend.controller;
+
+import de.unistuttgart.overworldbackend.data.PlayerTaskStatisticDTO;
+import de.unistuttgart.overworldbackend.data.PlayerTaskStatisticData;
+import de.unistuttgart.overworldbackend.service.PlayerTaskStatisticService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import javax.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@Tag(name = "Submit game data", description = "Submit a game statistic")
+@RestController
+@Slf4j
+@RequestMapping("/internal")
+public class MinigameInputController {
+
+  @Autowired
+  PlayerTaskStatisticService playerTaskStatisticService;
+
+  @Valid
+  @Operation(summary = "Input Data from a minigame")
+  @PostMapping("/submit-game-pass")
```

Comment:
```suggestion
  @PostMapping("/game-pass")
```
I think it isn't a good idea to include submit into the mapping. It is a post mapping so this is clear that it "submits" some data

Maybe we should also call it `minigame-pass` instaed of `game-pass` to make it more clearly that this is only about data from minigames

Reply:
will check after review in next sprint